bedrock/base/geo.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+from django.conf import settings
+
+try:
+    import maxminddb
+except ImportError:
+    maxminddb = None
+
+
+logger = logging.getLogger(__name__)
```

Comment:
FYI this causes a warning `No handlers could be found for logger "bedrock.base.geo"` for me locally